lib/basket.rb

```diff
@@ -0,0 +1,36 @@
+class Basket
+
+  attr_reader :dishes
+  
+  def initialize(menu)
+    @dishes = []
+    @menu = menu
+  end
+
+  def add(dish, quantity = 1)
+    @menu.search(dish)
+    quantity.times { @dishes << dish }
+  end  
+  
+  def remove(dish, quantity = 1)
```

Comment:
if you'er defaulting the quantity in the method that calls this method, I don't think you need to default it again!

- Furthermore, is there a reason to have a call for this inside the order class? or can it be called directly in IRB? order.remove calls basket.remove but could you not do basket.remove straight?